### PR TITLE
139 add some fields

### PR DESCRIPTION
**Pull request by rlecellier**

### CODE REVIEW

File: formbuilder/config.js [hunk 1]
```diff
@@ -99,5 +99,55 @@ export default {
       },
       formData: {}
     },
+    {
+      id: "select",
```
Comment (almet) [inline formbuilder/config.js]: Hi, the select has been removed a few months ago since it has the same use as the choice list. I'm not sure what we should do with this idea, since I personally like better this one, so maybe we should replace the former with the latter.

File: formbuilder/config.js [hunk 2]
```diff
@@ -99,5 +99,55 @@ export default {
...
+                type: "string"
+              }
+            }
+          }
+        },
+      },
+      formData: {}
+    },
+    {
+      id: "date",
```
Comment (almet) [inline formbuilder/config.js]: Good idea!

File: formbuilder/components/builder/EditableField.js [hunk 3]
```diff
@@ -55,7 +55,9 @@ class FieldPropertiesEditor extends Component {
             schema={uiSchema.editSchema}
             formData={formData}
             onChange={this.onChange.bind(this)}
-            onSubmit={onUpdate} />
+            onSubmit={onUpdate}>
+            <button type="submit" className="btn btn-info pull-right">Submit</button>
```
Comment (almet) [inline formbuilder/components/builder/EditableField.js]: Why did you need to add a button here?

Comment (rlecellier) [inline formbuilder/components/builder/EditableField.js]: In order to correct display of date widget, i had to remove a css rule .something . btn { float right}, this rule was used to pull right this submit button.
Instead of using default Form's submit button, i add one myself with 'pull-right' class.

Comment (almet) [inline formbuilder/components/builder/EditableField.js]: Gotcha.

**Comment by rlecellier** [comment]
if you look at who google does it, they seems having no concern about multiple way to represent data.
screenshot from 2016-09-19 18-59-01

**Comment by almet** [comment]
Yeah, this might not be a problem after all!

**Comment by almet** [comment]
Thank you for your work @rlecellier
